content/en/docs/home/contribute/localization.md

```diff
@@ -8,47 +8,87 @@ approvers:
+### Getting started
 
-**Note:** To find the most recent release's documentation source files:
-1. Navigate to https://github.com/kubernetes/website.
-2. Select the `release-1.X` branch for the most recent version.
+In order to add a new localization of the Kubernetes documentation, you'll need to make a few modifications to the site's [configuration](#configuration) and [directory structure](#new-directory), and then you can get to work [translating documents](#translating-documents)!
 
-    For example, the branch for Kubernetes v1.9 docs is `release-1.9`.
+We'll walk you through this whole process by adding support for the German language.
 
 {{< note >}}
-Source files reside in the `/docs/` directory.
+For an example localization-related [pull request](../create-pull-request) adding Korean localization to the Kubernetes docs, see [this pull request](https://github.com/kubernetes/website/pull/8636) to the [Kubernetes website repo](https://github.com/kubernetes/website).
 {{< /note >}}
 
-### Repository
+To get started, clone the website repo and `cd` into it:
 
-A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs-cn](https://github.com/kubernetes/kubernetes-docs-cn).
+```bash
+$ git clone https://github.com/kubernetes/website
+$ cd website
+```
 
-{{< note >}}
-**Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
-{{< /note >}}
+### Releases
+
+Localizations must use English files from the [most recent major
+release](https://kubernetes.io/docs/home/supported-doc-versions/#current-version) as sources.
+
+To find the most recent release's documentation source files:
+
+1. Navigate to the Kubernetes website repository at https://github.com/kubernetes/website.
+2. Select the `release-1.X` branch for the most recent version. The branch for Kubernetes v1.9 docs, for example, is `release-1.9`.
+
+### Configuration
+
+The website's configuration is in the [`config.toml`](https://github.com/kubernetes/website/tree/master/config.toml) file. You'll need to add a configuration block for the new language to that file, under the existing `[languages]` blocks. The German block looks like this:
+
+```toml
+[languages]
+[languages.de]
+title = "Kubernetes"
+description = "Produktionsreife Container-Verwaltung"
+languageName = "Deutsch"
+contentDir = "content/de"
+weight = 3
+```
 
-### Project
+When assigning a `weight` parameter, make sure to see which of the current languages has the highest weight and simply add 1.
+
+### New directory
+
+In order to begin adding documentation for a new localization, you'll need to add a subdirectory to the
+
+In the configuration example [above](#configuration), for example, the two-letter code for German is `de`. That means that a `de` directory needs to be added to the `content` directory:
```

Comment:
remove "for example, "

Reply:
Please note that this is a WIP and not yet ready for review